### ISSUE TITLE
Broken integration tests with new consensus 

### ISSUE THREAD

**Issue by platfowner** [ainblockchain/ain-blockchain#45]
It looks like the block generation is stuck by some reasons.

```
./node_modules/mocha/bin/mocha --timeout 320000 -R  spec ./integration/afan_client.test.js


  aFan Client Test
    tx_invest
Initial last block number: 0
      1) "before each" hook for "send_one"
Initial last block number: 0
      2) "after each" hook for "send_one"


  0 passing (11m)
  2 failing

  1) aFan Client Test
       "before each" hook for "send_one":
     Error: Timeout of 320000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/seo/git/comcomai/blockchain/ain-blockchain/integration/afan_client.test.js)
      at module.exports.runLoopOnce (node_modules/deasync/index.js:65:10)
      at module.exports (node_modules/deasync-promise/index.js:13:9)
      at module.exports.interval (node_modules/system-sleep/module.js:24:36)
      at waitForNewBlocks (integration/afan_client.test.js:93:7)
      at set_value.then (integration/afan_client.test.js:117:19)

  2) aFan Client Test
       "after each" hook for "send_one":
     Error: Timeout of 320000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/seo/git/comcomai/blockchain/ain-blockchain/integration/afan_client.test.js)
      at module.exports.runLoopOnce (node_modules/deasync/index.js:65:10)
      at module.exports (node_modules/deasync-promise/index.js:13:9)
      at module.exports.interval (node_modules/system-sleep/module.js:24:36)
      at waitForNewBlocks (integration/afan_client.test.js:93:7)
      at set_value.then (integration/afan_client.test.js:122:19)
      at process._tickCallback (internal/process/next_tick.js:68:7)
      at module.exports.runLoopOnce (node_modules/deasync/index.js:64:10)
      at module.exports (node_modules/deasync-promise/index.js:13:9)
      at module.exports.interval (node_modules/system-sleep/module.js:24:36)
      at waitForNewBlocks (integration/afan_client.test.js:93:7)
      at set_value.then (integration/afan_client.test.js:117:19)

```

**Comment by minsulee2** [comment]
@liayoo has it been updated issue, init?! You can close!

**Comment by liayoo** [comment]
Closing this issue as the problem has been fixed in #47.
